### PR TITLE
Bug/post show page design

### PR DESCRIPTION
**Pull request by ksowndar**
Post show page is designed

### CODE REVIEW

File: app/views/posts/_comments.html.erb [hunk 1]
```diff
@@ -1,35 +1,26 @@
...
-            <% if signed_in? %><a href = "#" class="comment-form-display">Reply</a><% end %>
-          <% end %>
-          <div class = "comment-form">
-            <%= render partial: "comments/form", locals: {post: comment.post, parent: comment, comment_id: comment.id}%>
-          </div>
-        </div>
+<div class = "comment">  
+  <% comments.each do |comment| %>
+    <% if comment.account.profile_image.attached? %>
+      <%= link_to image_tag(comment.account.profile_image, width: "30px" , height:"30px", :class => "small-pic float-left m-r-15"),profile_path(comment.account) %>
```
Comment (Venirails) [inline app/views/posts/_comments.html.erb:4]: still in some of the places this old syntax is used.please fix this

:class=>

File: app/views/comments/_reply.html.erb [hunk 2]
```diff
@@ -1,33 +1,24 @@
...
-          <% if reply.replies.any? %>
-            <% reply.replies.each do |reply| %>
-              <div class = "sub-comment"><%= render partial: 'comments/reply', locals: { reply: reply, comment_id: reply.id } %></div>
-            <% end %>
-          <% end %>
-        </div>
-      </div>
+<div class = "comment" >
+  <% if reply.account.profile_image.attached? %>
+    <%= link_to image_tag(reply.account.profile_image, width: "30px" , height:"30px", :class => "small-pic float-left m-r-15"),profile_path(reply.account.username) %>
```
Comment (Venirails) [inline app/views/comments/_reply.html.erb:3]: please change this also

Comment (ksowndar) [reply app/views/comments/_reply.html.erb]: It is changed

File: app/views/posts/_comments.html.erb (hunk 1 above)
Comment (ksowndar) [reply app/views/posts/_comments.html.erb]: I have changed the syntax

**Pull request merged**